lib/experimental/Information/Tags/BaseTag/index.tsx

```diff
@@ -14,24 +14,31 @@ type Props = {
 }
 
 export const BaseTag = forwardRef<HTMLDivElement, Props>(
-  ({ left, text, additionalAccesibleText, onClick, className }, ref) => (
-    <div
-      ref={ref}
-      className={cn(
-        "line-clamp-1 flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 text-base font-medium",
-        onClick && "cursor-pointer hover:bg-f1-background-hover",
-        !left ? "pl-2" : "pl-1",
-        className
-      )}
-      onClick={onClick}
-    >
-      {left}
-      <span>{text}</span>
-      {additionalAccesibleText && (
-        <span className="sr-only">, {additionalAccesibleText}</span>
-      )}
-    </div>
-  )
+  ({ left, text, additionalAccesibleText, onClick, className }, ref) => {
+    if (!text && !left) {
```

Comment:
let's not go this way, but provide two prop types:

```
type MandatoryLeftProps = { left: ReactNode, text?: string}
type MandatoryTextProps = { left?: ReactNode, text: string}

type Props = MandatoryLeftProps | MandatoryTextProps
```

If nothing is passed, it will [throw an error](https://www.typescriptlang.org/play/?#code/C4TwDgpgBAsghgOwCZ2AewE4gDIQGbAAKGaYAzlALxQDeUANvsAFxQICuAtgEYQYA0UYBAAewAPysywDAEsEAcwC+AKFCRYiFOiwAVUURLkqtBk0lsuvAUINSZ85SrXhoxUhWrxkqTDibuxgA+mj46IPpigWTOeOwIAMbAsmgIttIAFGBGZKzRAJSs3GhojIi0KlBQGBDA7BhpMuwQKkpQzsKZNIwErACsgsJizABEI0r5ahBdPSxQfRNTXUMsY4udwBk0i0A)

Reply:
Better way to express it definitely, thanks! Just commited it!